### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by musvaage**
_cf:_ https://github.com/zenodo/zenodo/pull/2476#issuecomment-1823529999

### CODE REVIEW

File: tests/unit/deposit/test_api_simpleflow.py [hunk 1]
```diff
@@ -458,7 +458,7 @@ def test_versioning_rest_flow(mocker, api, api_client, db, es, locations,
     )
     assert response.status_code == 201
 
-    # Publish new verision
+    # Publish new version
```
Comment (yarikoptic) [inline tests/unit/deposit/test_api_simpleflow.py:461]: cool, filed https://github.com/codespell-project/codespell/pull/3252 ;)

**Comment by musvaage** [comment]
@slint

What 's the explanation for the failing check?

> In retrospect if any of the description fields modified in `#2423` (Closed) should be reverted this can be pursued.

The description fields originated externally to this repository and factually the modifications thereto should be reverted.

**Comment by musvaage** [comment]
it was incorrect to approve that the description fields be modified in `#2423` (Closed)

that being said there neither appears to be forthcoming Maintainer response on this pull nor a decision on `#2470`

**Comment by yarikoptic** [comment]
if "maintainer" is to be me -- please let me know what is desired to be done here??? 

Frankly I am surprised that this PR with such trivial fixes requires some more discussion and was not yet merged.

**Comment by musvaage** [comment]
The <kbd>Author</kbd>, <kbd>Maintainer</kbd> or  <kbd>Member</kbd> may identify the relation of the individual to a ticket.

**Comment by musvaage** [comment]
> it was incorrect to approve that the description fields be modified in `#2423` (Closed)

Notwithstanding other Maintainer priorities, but I'm reluctant to even open a PR reverting those incorrect changes in `#2423`.

**Review by yarikoptic — APPROVED** [review]

**Comment by yarikoptic** [comment]
must be me but I am not following. Anyways -- apparently I was assigned a reviewer, I approve . CI fails since python 2.7 is no more (so someone should address that separately).  Let me know if I could be of any other help.

**Comment by slint** [comment]
@yarikoptic @musvaage I'm sorry to jump in this late. This repository has been somewhat inactive for the past year or so, because we've moved development over to https://github.com/zenodo/zenodo-rdm/ due to a large upgrade and migration to Zenodo. We plan to transfer the code back to this repository and continue development here, but there's no specific timeline for that. 

In the meantime, I will look into how it would be possible to notify/warn that PRs in this repository's current state should be avoided.

Thank you for your contributions, but unfortunately I'll have to close this PR and https://github.com/zenodo/zenodo/pull/2470.

**Comment by ppanero** [comment]
@slint you can use [`peter-evans/create-or-update-comment@v3`](https://github.com/peter-evans/create-or-update-comment) GH action to automatically add comments to PRs

**Comment by musvaage** [comment]
> description fields be modified in `#2423`

Again, so it's not overlooked when development starts up again in this repo, those description fields were from external sources.

> reviewer

A reviewer to a GH pull would typically be a repo <kbd>Collaborator</kbd>, <kbd>Maintainer</kbd> or  <kbd>Member</kbd>, that is someone with write authority.

**Pull request closed without merging**